service/debugger/debugger.go

```diff
@@ -2426,3 +2429,51 @@ var attachErrorMessage = attachErrorMessageDefault
 func attachErrorMessageDefault(pid int, err error) error {
 	return fmt.Errorf("could not attach to pid %d: %s", pid, err)
 }
+
+func (d *Debugger) maybePrintUnattendedBreakpointWarning(currentThread *api.Thread, clientStatusCh <-chan struct{}) {
+	select {
+	case <-clientStatusCh:
+		// the channel will be closed if the client that sends the command has left
+		// i.e. closed the connection.
+	default:
+		return
+	}
+
+	const defaultStackTraceDepth = 50
+	frames, err := d.stacktrace(currentThread.GoroutineID, defaultStackTraceDepth, 0)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "err", err)
+		return
+	}
+
+	apiFrames, err := d.convertStacktrace(frames, nil)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "err", err)
+		return
+	}
+
+	bp := currentThread.Breakpoint
+	if bp == nil {
+		fmt.Fprintln(os.Stderr, "bp", bp)
```

Comment:
i don't think this log line is necessary. When attaching a debugger to a long running process we get `bp <nil>` logged out. It'll never be more useful than that.

Reply:
@fatanugraha